differ/differ.go

```diff
@@ -301,6 +309,97 @@ func evaluateFilterExpression(eventFilter string, thresholds EventThresholds, ev
 	return evaluator.Evaluate(eventFilter, values)
 }
 
+func findRenderedReport(reports []types.RenderedReport, ruleName types.RuleName, errorKey types.ErrorKey) (types.RenderedReport, error) {
+	for _, report := range reports {
+		reportRuleName := string(report.RuleID)[strings.LastIndex(string(report.RuleID), ".")+1:]
+		if reportRuleName == string(ruleName) && report.ErrorKey == errorKey {
+			return report, nil
+		}
+	}
+	return types.RenderedReport{}, errors.New("report for given rule ID nad error key has not been found")
+}
+
+func createServiceLogEntry(report types.RenderedReport, cluster types.ClusterEntry) types.ServiceLogEntry {
+	logEntry := types.ServiceLogEntry{
+		ClusterUUID: string(cluster.ClusterName),
+		Description: report.Description,
+		ServiceName: "CCX Notification Service",
```

Comment:
I guess we could define a constant with "CCX Notification Service" as value, like all the other strings

Reply:
Replaced with constant